.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
This file include a list of file to _exclude_ from `git` watching.  I use an editor called Vim which creates temporary files while I'm editing that end in `.swp`.  This makes it so I won't accidentally commit files that shouldn't be here.